tests/functional/aws-node-sdk/test/versioning/multiObjectDelete.js

```diff
@@ -138,3 +139,96 @@ testing('Multi-Object Versioning Delete Success', function success() {
         });
     });
 });
+
+testing('Multi-Object Versioning Delete - deleting delete marker',
+() => {
+    withV4(sigCfg => {
+        const bucketUtil = new BucketUtility('default', sigCfg);
+        const s3 = bucketUtil.s3;
+
+        beforeEach(done => {
+            async.waterfall([
+                next => s3.createBucket({ Bucket: bucketName },
+                    err => next(err)),
+                next => s3.putBucketVersioningAsync({
+                    Bucket: bucketName,
+                    VersioningConfiguration: {
+                        Status: 'Enabled',
+                    },
+                }, err => next(err)),
+            ], done);
+        });
+        afterEach(done => {
+            removeAllVersions({ Bucket: bucketName }, err => {
+                if (err) {
+                    return done(err);
+                }
+                return s3.deleteBucket({ Bucket: bucketName }, err => {
+                    assert.strictEqual(err, null,
+                        `Error deleting bucket: ${err}`);
+                    return done();
+                });
+            });
+        });
+
+        it('deleteing a delete marker - should send back VersionId and ' +
+        'DeleteMarkerVersionId equals to deleteVersionId', done => {
```

Comment:
equals should be 'equal'

also, would be clearer to say 'should send back VersionId and DeleteMarkerVersionId both equal to deleteVersionId'.